### ISSUE TITLE
libsmartcols: bitwise operations with 0

### ISSUE THREAD

**Issue by ignatenkobrain** [karelzak/util-linux#385]
SCOLS_CELL_FL_LEFT is constant defined to `0`. Or we need to fix table printing code, or we need to fix SCOLS_CELL_FL_*, not sure what is better...

```c
diff --git a/libsmartcols/src/table_print.c b/libsmartcols/src/table_print.c
index 2aff9d181..0e667ffc2 100644
--- a/libsmartcols/src/table_print.c
+++ b/libsmartcols/src/table_print.c
@@ -782,11 +784,11 @@ static int print_title(struct libscols_table *tb)
                goto done;
        }
 
-       if (tb->title.flags & SCOLS_CELL_FL_LEFT)
+       if (tb->title.flags == SCOLS_CELL_FL_LEFT)
                align = MBS_ALIGN_LEFT;
-       else if (tb->title.flags & SCOLS_CELL_FL_RIGHT)
+       else if (tb->title.flags == SCOLS_CELL_FL_RIGHT)
                align = MBS_ALIGN_RIGHT;
-       else if (tb->title.flags & SCOLS_CELL_FL_CENTER)
+       else if (tb->title.flags == SCOLS_CELL_FL_CENTER)
                align = MBS_ALIGN_CENTER;
        else
                align = MBS_ALIGN_LEFT; /* default */
```

**Comment by ignatenkobrain** [comment]
note that for last line it's probably more correct to do something like `g_assert_not_reached()` (replace with some non-glib function)

**Comment by karelzak** [comment]
Ah, good catch. 

Should be fixed by commit 6ffbd28359104c75015c260b84c804fb893b8620, it implements new function scols_cell_get_alignment() to hide the bitops mess.
